### PR TITLE
Fixed hearthbeat bug 

### PR DESCRIPTION
**Pull request by lmarini**
where if the owner was not set in extractors_info.json clowder would throw and error and the extractor would not get registered.

### CODE REVIEW

**Comment by lmarini** [comment]
@max-zilla can you test a private extractor that includes an owner in extractor_info.json. Thank you.

File: version.sh [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 
-VERSION="1.22.0"
+VERSION="1.22.1"
 
 sed -i~ "s#^  val version = \".*\"\$#  val version = \"${VERSION}\"#" project/Build.scala
-sed -i~ "s#^version: .*\$#version: \"${VERSION}\"#" citation.cff
```
Comment (robkooper) [inline version.sh:6]: why not update the citation file?

**Review by max-zilla — APPROVED** [review]
all seems to work as expected.

**Comment by minump** [comment]
I was able to test this as well. Ran `ghcr.io/clowder-framework/clowder:PR-429`, deleted a private extractor from UI (http://localhost:8000/extractors/pdfminer-extractor) . Saw that the extractor came back on (name popped up in the list of extractors) after heartbeat time.

Tested. works fine. approving.

**Comment by lmarini** [comment]
> I was able to test this as well. Ran `ghcr.io/clowder-framework/clowder:PR-429`, deleted a private extractor from UI (http://localhost:8000/extractors/pdfminer-extractor) . Saw that the extractor came back on (name popped up in the list of extractors) after heartbeat time.
> 
> Tested. works fine. approving.

Thanks! Next time feel free to add yourself to list of reviewers.

**Pull request merged**